spec/ddtrace/workers/loop_spec.rb

```diff
@@ -64,7 +67,7 @@
 
       context 'when the worker is not running' do
         before { worker.stop_loop }
-        it { is_expected.to be false }
+        it { is_expected.to be true }
```

Comment:
`#stop_loop` now always returns `true`, as we can't know for sure if the stop signal has already been consumed by the main loop or not.

Reply:
Minor: Since we changed the spec on this method to always return true, perhaps we should just move this test to the parent context -- since it always applies, regardless of the state of the worker?